src/Calculator/App.xaml.cs

```diff
@@ -0,0 +1,533 @@
+                        }
+                    }
+                    // Set the preLaunched flag to false
+                    m_preLaunched = false;
+                }
+                else // for touch devices
+                {
+                    if (rootFrame.Content == null)
+                    {
+                        // When the navigation stack isn't restored navigate to the first page,
+                        // configuring the new page by passing required information as a navigation
+                        // parameter
+                        if (!rootFrame.Navigate(typeof(MainPage), argument))
+                        {
+                            // We couldn't navigate to the main page,
+                            // kill the app so we have a good stack to debug
+                            throw new SystemException();
+                        }
+                    }
+                    if (ApplicationView.GetForCurrentView().ViewMode != ApplicationViewMode.CompactOverlay)
+                    {
+                        if (!Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.Phone.UI.Input.HardwareButtons"))
+                        {
+                            // for tablet mode: since system view activation policy is disabled so do ShowAsStandaloneAsync if activationViewSwitcher exists in
+                            // activationArgs
+                            ActivationViewSwitcher activationViewSwitcher = null;
+                            var activateEventArgs = (args as IViewSwitcherProvider);
+                            if (activateEventArgs != null)
+                            {
+                                activationViewSwitcher = activateEventArgs.ViewSwitcher;
+                            }
+                            if (activationViewSwitcher != null)
+                            {
+                                var viewId = (args as IApplicationViewActivatedEventArgs).CurrentlyShownApplicationViewId;
+                                if (viewId != 0)
+                                {
+                                    _ = activationViewSwitcher.ShowAsStandaloneAsync(viewId);
+                                }
+                            }
+                        }
+                        // Ensure the current window is active
+                        Window.Current.Activate();
+                    }
+                }
+            }
+        }
+
+        private void DismissedEventHandler(SplashScreen sender, Object e)
+        {
+            _ = SetupJumpList();
+        }
+
+        private void RegisterDependencyProperties()
+        {
+            NarratorNotifier.RegisterDependencyProperties();
+        }
+
+        private void OnSuspending(Object sender, SuspendingEventArgs args)
+        {
+            // CSHARP_MIGRATION: TODO:
+            //TraceLogger.GetInstance().LogButtonUsage();
```

Comment:
This one is available now, we can uncomment it.

Reply:
Uncommented.